---
title: CV
position: 1
---

> Work in progress :construction_worker:

## Experience

### European Bioinformatics Institute (EMBL-EBI), Hinxton, United Kingdom

I work in the Comparative Genomics team of the Ensembl genome browser. We are in charge of
comparing the genomes to one another, implementing new methods and algorithms
(extending our API and database schema), and applying them on new datasets. Scalability is
our main focus as we have to process hundreds (soon thousands !) of genomes in a limited timescale.

I am also involved in the development of the eHive workflow manager, a system for creating
and running workflows on a distributed compute resource. It is now responsible for
scheduling and executing in excess of 1,000 CPU years of compute per year in Ensembl.

#### Principal Developer

_Since Oct 2019_

Development management. Technical leadership. Recruitment. Mentorship.
API development. Database design and optimisation. Workflow design and development. 
User support (data, API, workflows). Team size: 3 people.

In this role, I transfer the knowledge I have accumulated in the past 8 years
to the new Project Leader and the developers, whilst helping them and
overseeing the development of the software.

We are undertaking a massive revamp of our compute workflows and data storage
strategy, in order to cope with the scale of data that projects such as the
Darwin Tree of Life will generate. Our aim is ambitious: provide comparative
analyses on tens of thousands of genomes, and more.

I still maintain and contribute to the development of the eHive workflow manager,
keeping it the most efficient solution for the Ensembl Comparative Genomics workflows.

#### Project Leader

_May 2014 to Sep 2019._

Project planning and management. Scientific and public communication. Technical leadership. Recruitment. Mentorship.
Development management. Reporting. Data-production planning and operational management.
API development. Database design and optimisation. Workflow design and development. Data
production under tight deadlines. Processing of large datasets. User support (data, API, workflows).
Team size: 2-5 people.

I managed the whole Comparative Genomics team of the Ensembl project,
incl. the development of the eHive workflow manager.
We had to provide technical support to other Ensembl teams
who were using our software. During this period, we wrote an extensive user manual
for eHive.

#### Interim Manager

_May 2013 to May 2014._

Development management. Reporting. Technical advisor. Data-production planning and operational management.
API development. Database design and optimisation. Workflow design and development. Data
production under tight deadlines. Processing of large datasets. User support (data, API, workflows).
Team size: 2 people.

I was managing part of the Comparative Genomics team of
Ensembl, incl. the development of the eHive workflow manager,
and was still carrying on my developer duties. Our work was focused on
the reconstruction of phylogenetic trees and gene families, improving and
extending the software. I was still giving some Ensembl API workshops.

#### Software Developer

_Jan 2011 to May 2013._

API development. Database design and optimisation. Workflow design and development. Data
production under tight deadlines. Processing of large datasets. User support (data, API, workflows).

I focused on the reconstruction of protein phylogenetic trees, reshaping the
API, improving and extending the software. I also gave some Ensembl API
workshops.

## IT skills

Programming languages and technologies I know and use (or have used):

|---|---|---|
| Perl | <img src="https://raw.githubusercontent.com/github/explore/master/topics/perl/perl.png" alt="Perl" height="24"> | ![4 &#x2605;]({{ site.url }}/assets/img/4.svg) |
| Python | <img src="https://raw.githubusercontent.com/github/explore/master/topics/python/python.png" alt="Python" height="24"> | ![4 &#x2605;]({{ site.url }}/assets/img/4.svg) |
| Java | <img src="https://raw.githubusercontent.com/github/explore/master/topics/java/java.png" alt="Java" height="24"> | ![3 &#x2605;]({{ site.url }}/assets/img/3.svg) |
| C | <img src="https://raw.githubusercontent.com/github/explore/master/topics/c/c.png" alt="C" height="24"> | ![3 &#x2605;]({{ site.url }}/assets/img/3.svg) |
| C++ | <img src="https://raw.githubusercontent.com/github/explore/master/topics/cpp/cpp.png" alt="C++" height="24"> | ![3 &#x2605;]({{ site.url }}/assets/img/3.svg) |
| R | <img src="https://raw.githubusercontent.com/github/explore/master/topics/r/r.png" alt="R" height="24"> | ![2 &#x2605;]({{ site.url }}/assets/img/2.svg) |
| Go | <img src="https://raw.githubusercontent.com/github/explore/master/topics/go/go.png" alt="Go" height="24"> | ![1 &#x2605;]({{ site.url }}/assets/img/1.svg) |
| Ocaml | <img src="https://raw.githubusercontent.com/github/explore/master/topics/ocaml/ocaml.png" alt="Ocaml" height="24"> | ![1 &#x2605;]({{ site.url }}/assets/img/1.svg) |
| PHP | <img src="https://raw.githubusercontent.com/github/explore/master/topics/php/php.png" alt="R" height="24"> | ![1 &#x2605;]({{ site.url }}/assets/img/1.svg) |
| MySQL | <img src="https://raw.githubusercontent.com/github/explore/master/topics/mysql/mysql.png" alt="MySQL" height="24"> | ![4 &#x2605;]({{ site.url }}/assets/img/4.svg) |
| SQLite | <img src="https://raw.githubusercontent.com/github/explore/master/topics/sqlite/sqlite.png" alt="SQLite" height="24"> | ![3 &#x2605;]({{ site.url }}/assets/img/3.svg) |
| Bash | <img src="https://raw.githubusercontent.com/github/explore/master/topics/bash/bash.png" alt="Bash" height="24"> | ![5 &#x2605;]({{ site.url }}/assets/img/5.svg) |
| Docker | <img src="https://raw.githubusercontent.com/github/explore/master/topics/docker/docker.png" alt="Docker" height="24"> | ![4 &#x2605;]({{ site.url }}/assets/img/4.svg) |
| Git | <img src="https://raw.githubusercontent.com/github/explore/master/topics/git/git.png" alt="Git" height="24"> | ![5 &#x2605;]({{ site.url }}/assets/img/5.svg) |
