## About

I am the Principal Developer of the [comparative
genomics](http://www.ensembl.org/info/docs/compara/index.html) team (aka
Compara) of the [Ensembl](http://www.ensembl.org) project, which
includes the development of the [TreeFam](http://treefam.org/) project.
I also routinely contribute to
[eHive](https://ensembl-hive.readthedocs.io), the workflow manager that
powers all the production work in Ensembl.

I've got some experience in:

* Computational biology, Genome analysis.
* Comparative genomics, Evolution, Phylogenetics.
* Genome assembly and annotation
* Sequence alignment, Sequence classification, Homology inference.
* Algorithmics, Graph theory, Code optimization.
* Algorithms for 3D rendering.

## Topics of interest

- &#x1F52D; Reconstruction of protein phylogenetic trees and inference of
  orthologies
* &#x1F52E; Reconstruction of ancestral genomes
* &#x1F333; Tree of Life projects
* &#x1F465; Whole genome duplications
* &#x26A1; High-throughput computing

## Publications

My ORCID iD is
[![](https://orcid.org/sites/default/files/images/orcid_16x16.png)
0000-0002-7860-3560](http://europepmc.org/authors/0000-0002-7860-3560).
My publications ([Europe PubMed
Central](http://europepmc.org/search?query=AUTHORID:0000-0002-7860-3560&sortby=Date),
[Google Scholar](https://scholar.google.com/citations?user=CAik9V8AAAAJ),
[Scopus](https://www.scopus.com/authid/detail.uri?authorId=23485883100))
fall in these five topics:

* [Ensembl](http://www.ensembl.org) and [TreeFam](http://www.treefam.org)
* [Quest for Orthologues](https://questfororthologs.org)
* Genome analysis
* [Genomicus](http://www.dyogen.ens.fr/genomicus/) and ancestral-genome reconstruction
* Research in comparative genomics

## Background

I graduated in 2006 from both the [Institut d\'Informatique
d\'Entreprise](http://www.ensiie.fr) and the [Université
d\'Évry](http://www.univ-evry.fr/). I completed in 2010 a PhD in
bioinformatics in the [Dyogen
group](http://www.biologie.ens.fr/dyogen/?lang=en) under the supervision
of Hugues Roest Crollius, reconstructing the gene order and the
chromosome structure of extinct vertebrate species. The method was
validated using simulations, and the results are available on the
[Genomicus](http://www.dyogen.ens.fr/genomicus/) browser and updated
with the Ensembl releases. The algorithms are described in my
[thesis](http://hal.archives-ouvertes.fr/docs/00/55/21/38/PDF/these.pdf)
(in French).

I have joined the comparative-genomics team of the [Ensembl
project](http://www.ensembl.org) (aka Compara) in January 2011, under
the leadership of Javier Herrero. My initial focus was the pipeline that
builds protein-trees for the Ensembl release, but I have since worked on
every aspect of the Perl API and every pipeline, and I have gained
extensive knowledge of the eHive workflow manager. I led the team from
Javier\'s departure in 2013 to 2019, and now act as a Principal Developer,
overseeing and guiding the software development.

## Contact

Various profile pages with contact details &#x1F44B;:

* my [EBI profile](https://www.ebi.ac.uk/about/people/matthieu-muffato),
* my [<img
  src="https://static-exp1.licdn.com/scds/common/u/images/logos/favicons/v1/favicon.ico"
  alt="LinkedIn" height="16"> LinkedIn profile](https://uk.linkedin.com/in/matthieu-muffato),
* On GitHub, I am [<img src="https://raw.githubusercontent.com/github/explore/master/topics/github/github.png" alt="GitHub" height="16"> muffato](https://github.com/muffato).

Write me in &#x1F1EC;&#x1F1E7; English or in &#x1F1EB;&#x1F1F7; French.
Unfortunately, my notions of &#x1F1EA;&#x1F1F8; Spanish and
&#x1F1F0;&#x1F1F7; Korean are too limited for professional settings.

