---
title: About
position: 1
---

## About

I am the Informatics Infrastructure Team Lead of the [Tree of Life
programme](https://www.sanger.ac.uk/programme/tree-of-life/) of the
[![icon](/assets/img/icon/sanger.png) Wellcome Sanger Institute](https://www.sanger.ac.uk/).
My team oversees the implementation and delivery of the genome assembly pipelines,
and provides support for large-scale genome analyses for the faculty teams.

My experience is in:

* Software development
* Computational biology, Genome analysis.
* Comparative genomics, Evolution, Phylogenetics.
* Genome assembly and annotation
* Sequence alignment, Sequence classification, Homology inference.
* Algorithmics, Graph theory, Code optimization.
* Algorithms for 3D rendering.
* Project management

And I love:

* &#x1F52D; Reconstruction of protein phylogenetic trees and inference of
  orthologies
* &#x1F52E; Reconstruction of ancestral genomes
* &#x1F333; Tree of Life projects
* &#x1F465; Whole genome duplications
* &#x26A1; High-throughput computing

## Background

I graduated in 2006 from both the [![icon](/assets/img/icon/ensiie.png) Institut d\'Informatique
d\'Entreprise](http://www.ensiie.fr) and the [![icon](/assets/img/icon/evry.png) Université
d\'Évry](http://www.univ-evry.fr/). I completed in 2010 a PhD in
bioinformatics in the [Dyogen
group](http://www.biologie.ens.fr/dyogen/?lang=en) under the supervision
of Hugues Roest Crollius, reconstructing the gene order and the
chromosome structure of extinct vertebrate species. The method was
validated using simulations, and the results are available on the
[![icon](/assets/img/icon/genomicus.png) Genomicus](http://www.dyogen.ens.fr/genomicus/) browser and updated
with the Ensembl releases. The algorithms are described in my
[&#x1F393; thesis](http://hal.archives-ouvertes.fr/docs/00/55/21/38/PDF/these.pdf)
(in French).

I have joined the comparative-genomics team of the [![icon](/assets/img/icon/ensembl.png) Ensembl
project](http://www.ensembl.org) (aka Compara) in January 2011, under
the leadership of Javier Herrero. My initial focus was the pipeline that
builds protein-trees for the Ensembl release, but I have since worked on
every aspect of the Perl API and every pipeline, and I have gained
extensive knowledge of the ![icon](/assets/img/icon/guihive.png) eHive workflow
manager. I have given a number of [API workshops](https://www.ebi.ac.uk/~muffato/workshops/) too. I led the team from
Javier\'s departure in 2013 to 2019, and acted as a Principal Developer,
overseeing and guiding the software development, during a transition phase
with the new management.

Since 2021, I am at the [![icon](/assets/img/icon/sanger.png) Wellcome Sanger Institute](https://www.sanger.ac.uk/).
My ambition is to provide the most efficient platform for assembling and
analysing genomes at an unprecedented scale. The Tree of Life projects will
generate tens of thousands of high-quality genomes over the coming years
&mdash;
more than have ever been sequenced ! It is a challenging and extremely
exciting task that will shape the future of biology.

We are the interface between the Tree of Life teams (assembly production
and faculty research) and the Sanger's IT teams, working together with the
informatics teams of the other programmes. The work involves a wide range
of scientific fields and technologies such as assembly methods, genomics,
comparative genomics, cloud computing, large-scale analyses, with a strong
emphasis on metadata tracking, quality controls, and event recording.

## Contact

Various profile pages with contact details &#x1F44B;:

* my [![icon](/assets/img/icon/sanger.png) Sanger profile](https://www.sanger.ac.uk/person/muffato-matthieu/),
* my [![icon](/assets/img/icon/linkedin.png) LinkedIn profile](https://uk.linkedin.com/in/matthieu-muffato),
* my [![icon](/assets/img/icon/github.png) GitHub page](https://github.com/muffato).

Write me in &#x1F1EC;&#x1F1E7; English or in &#x1F1EB;&#x1F1F7; French.
Unfortunately, my notions of &#x1F1EA;&#x1F1F8; Spanish and
&#x1F1F0;&#x1F1F7; Korean are too limited for professional settings.

