---
title: About
position: 1
---

## About

I lead the [Informatics Infrastructure team](https://www.sanger.ac.uk/group/tree-of-life-informatics-infrastructure/)
I lead the [Informatics Infrastructure team](https://www.sanger.ac.uk/group/tree-of-life-informatics-infrastructure/)
for the [![icon](/assets/img/icon/tol.png) Tree of Life programme](https://www.sanger.ac.uk/programme/tree-of-life/)
at the [![icon](/assets/img/icon/sanger.png) Wellcome Sanger Institute](https://www.sanger.ac.uk/).

The team's mission is to _enable Tree of Life operations and research by
providing data management, bioinformatics, and infrastructure solutions_.
We implement large-scale genome analysis pipelines for the faculty teams,
maintain the production environment (application deployment and pipeline orchestration),
and manage and curate metadata across our production systems.

My experience is in:

* Software development
* Project management and planning
* Computational biology and genome analysis
* Comparative genomics, evolution, phylogenetics
* Sequence alignment, sequence classification, homology inference
* Algorithmics and graph theory

I love:

* &#x1F52D; Reconstruction of phylogenetic trees and inference of
  orthologies
* &#x1F52E; Reconstruction of ancestral genomes
* &#x1F333; Tree of Life projects
* &#x1F465; Whole-genome duplications
* &#x26A1; High-throughput computing
* &#x1F9CA; Algorithms for 3D rendering
* &#x1F916; Code optimisation and assembly

## Background

I graduated in 2006 from both the [![icon](/assets/img/icon/ensiie.png) Institut d\'Informatique
d\'Entreprise](http://www.ensiie.fr) and the [![icon](/assets/img/icon/evry.png) Université
d\'Évry](http://www.univ-evry.fr/). I completed in 2010 a PhD in
bioinformatics with the [Dyogen group](http://www.biologie.ens.fr/dyogen/?lang=en) under the supervision of Hugues Roest Crollius, where I reconstructed gene order and chromosome structure of extinct vertebrate species. The method was validated with simulations; the results are available on the [![icon](/assets/img/icon/genomicus.png) Genomicus](http://www.dyogen.ens.fr/genomicus/) browser and are updated with Ensembl releases. The algorithms are described in my [&#x1F393; thesis](http://hal.archives-ouvertes.fr/docs/00/55/21/38/PDF/these.pdf) (in French) and in this [&#x1F4D6; Nature Ecology & Evolution paper](https://doi.org/10.1038/s41559-022-01956-z).

I joined the comparative-genomics team of the [![icon](/assets/img/icon/ensembl.png) Ensembl project](http://www.ensembl.org) (Compara) in 2011 under the leadership of Javier Herrero. My initial focus was the pipeline that builds protein trees for Ensembl releases, but I later worked on all aspects of the Perl API and its pipelines. I developed deep expertise in the ![icon](/assets/img/icon/guihive.png) eHive workflow manager and became a co-developer.

I led the team from Javier's departure in 2013 until 2019, overseeing automation of our production processes. In my final year I supported the transition to the next project leader as Principal Developer, continuing to oversee software development.

I joined the [![icon](/assets/img/icon/sanger.png) Wellcome Sanger Institute](https://www.sanger.ac.uk/) in 2021 to create the Informatics Infrastructure team for the [![icon](/assets/img/icon/tol.png) Tree of Life programme](https://www.sanger.ac.uk/programme/tree-of-life/). I helped shape the team into its current form.

## Contact

Various profile pages with contact details &#x1F44B;:

* my [![icon](/assets/img/icon/sanger.png) Sanger profile](https://www.sanger.ac.uk/person/muffato-matthieu/),
* my [![icon](/assets/img/icon/linkedin.png) LinkedIn profile](https://uk.linkedin.com/in/matthieu-muffato),
* my [![icon](/assets/img/icon/github.png) GitHub page](https://github.com/muffato).

Send me a message in &#x1F1EC;&#x1F1E7; English or in &#x1F1EB;&#x1F1F7; French.
Feel free to message me in English or French. My Spanish and Korean are limited for professional use.
