---
title: Posters
position: 5
---

# Posters

The complete list of posters is available at these databases:

* [![icon](/assets/img/icon/f1000.png) F1000](https://f1000research.com/search?q=matthieu+muffato).
* [![icon](/assets/img/icon/zenodo.png) Zenodo](https://zenodo.org/search?q=metadata.creators.person_or_org.identifiers.identifier%3A%220000-0002-7860-3560%22&f=resource_type%3Aposter).

Find below a selection organised by themes:

* [![icon](/assets/img/icon/ensembl.png) Ensembl and ![icon](/assets/img/icon/treefam.png) TreeFam methods](#ENSEMBL)
* [![icon](/assets/img/icon/guihive.png) eHive workflow manager](#EHIVE)
* [![icon](/assets/img/icon/tol.png) Tree of Life Informatics Infrastructure](#TOLIT)

## ![icon](/assets/img/icon/ensembl.png) Ensembl and ![icon](/assets/img/icon/treefam.png) TreeFam methods {#ENSEMBL}

<dl>
<dt>Automated filtering of multiple sequence alignment worsens phylogenetic inference</dt>
<dd>
Ge Tan, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1115083.1](https://doi.org/10.7490/f1000research.1115083.1)
</dd>
<dt>TreeFam and Ensembl: Phylogenetic resources</dt>
<dd>
Mateus Patricio, Matthieu Muffato, Bronwen Aken, Paul Flicek \\
DOI: [10.7490/f1000research.1112451.1](https://doi.org/10.7490/f1000research.1112451.1)
</dd>
<dt>Ensembl comparative genomics update – HMMs and Orthology QC</dt>
<dd>
Mateus Patricio, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1114127.1](https://doi.org/10.7490/f1000research.1114127.1)
</dd>
<dt>Genome-wide assessment of ortholog quality in Ensembl</dt>
<dd>
Wasiu Akanni, Mateus Patricio, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1115084.1](https://doi.org/10.7490/f1000research.1115084.1)
</dd>
<dt>Scaling up the TreeFam resource in Ensembl</dt>
<dd>
Mateus Patricio, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1115011.1](https://doi.org/10.7490/f1000research.1115011.1)
</dd>
<dt>Moving towards comparative analysis of hundreds of vertebrate genomes</dt>
<dd>
Carla Cummins, Mateus Patricio, Wasiu Akanni, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1115012.1](https://doi.org/10.7490/f1000research.1115012.1)
</dd>
<dt>Inclusion of pseudogenes in the Ensembl comparative genomics resources</dt>
<dd>
Guillaume Giroussens, Matthieu Muffato, Paul Flicek \\
DOI: [10.7490/f1000research.1116155.1](https://doi.org/10.7490/f1000research.1116155.1)
</dd>
<dt>Scalable infrastructure for comparative genomics in the era of large-scale Tree of Life projects</dt>
<dd>
Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1118390.1](https://doi.org/10.7490/f1000research.1118390.1)
</dd>
</dl>

## ![icon](/assets/img/icon/guihive.png) eHive workflow manager {#EHIVE}

<dl>
<dt>eHive – Flexible and scalable pipelines</dt>
<dd>
Andy Yates, Bronwen Aken, Stephen Fitzgerald, Leo Gordon, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1111870.1](https://doi.org/10.7490/f1000research.1111870.1)
</dd>
</dl>

## ![icon](/assets/img/icon/tol.png) Tree of Life Informatics Infrastructure {#TOLIT}

<dl>
<dt>Core resources for genome analysis</dt>
<dd>
Matthieu Muffato, Mark Blaxter \\
DOI: [10.5281/zenodo.806387](https://doi.org/10.5281/zenodo.8063873)
</dd>
<dt>WESkit: Workflow execution for HTC and cloud</dt>
<dd>
Philip R. Kensche, Valentin Schneider-Lunitz, Cibin Sadasivan Baby, Alexander Kanitz, Matthieu Muffato, _et al._ \\
DOI: [10.7490/f1000research.1119541.1](https://doi.org/10.7490/f1000research.1119541.1)
</dd>
<dt>Sharing Nextflow code across organizations using the nf-core infrastructure</dt>
<dd>
João Vitor Ferreira Cavalcante, The nf-core community, Tree of Life Informatics Infrastructure Team, Matthieu Muffato \\
DOI: [10.5281/zenodo.13913054](https://doi.org/10.5281/zenodo.13913054)
</dd>
</dl>

