---
title: Publications
position: 4
---

# Publications

My ORCID iD is
[![icon](/assets/img/icon/orcid.png)
0000-0002-7860-3560](http://europepmc.org/authors/0000-0002-7860-3560).
Find the complete list of publications at [![icon](/assets/img/icon/epmc.png) Europe PubMed
Central](http://europepmc.org/search?query=AUTHORID:0000-0002-7860-3560&sortby=Date),
[![icon](/assets/img/icon/googlescholar.png) Google Scholar](https://scholar.google.com/citations?user=CAik9V8AAAAJ),
and [![icon](/assets/img/icon/scopus.png) Scopus](https://www.scopus.com/authid/detail.uri?authorId=23485883100).


## ![icon](/assets/img/icon/ensembl.png) Ensembl Comparative Genomics

<dl>
<dt>ncRNA orthologies in the vertebrate lineage.</dt>
<dd>
Pignatelli M, Vilella AJ, Muffato M, Gordon L, White S, Flicek P, Herrero J. \\
_Database : the journal of biological databases and curation_ 2016, 2016
</dd>
<dt>Ensembl comparative genomics resources.</dt>
<dd>
Herrero J, Muffato M, _et al._ \\
_Database : the journal of biological databases and curation_ 2016, 2016
</dd>
<dt>Current Methods for Automated Filtering of Multiple Sequence Alignments Frequently Worsen Single-Gene Phylogenetic Inference.</dt>
<dd>
Tan G, Muffato M, Ledergerber C, Herrero J, Goldman N, Gil M, Dessimoz C. \\
_Systematic biology_ 64(5):778-791, 2015 Sep
</dd>
<dt>TreeFam v9: a new website, more species and orthology-on-the-fly.</dt>
<dd>
Schreiber F, Patricio M, Muffato M, Pignatelli M, Bateman A. \\
_Nucleic acids research_ 42(Database issue):D922-5, 2014 Jan
</dd>
</dl>

## ![icon](/assets/img/icon/genomicus.png) Genomicus and ancestral-genome reconstruction

<dl>
<dt>The 3D organization of chromatin explains evolutionary fragile genomic regions.</dt>
<dd>
Berthelot C, Muffato M, Abecassis J, Roest Crollius H. \\
_Cell reports_ 10(11):1913-1924, 2015 Mar
</dd>
<dt>Genomicus update 2015: KaryoView and MatrixView provide a genome-wide perspective to multispecies comparative genomics.</dt>
<dd>
Louis A, Nguyen NT, Muffato M, Roest Crollius H. \\
_Nucleic acids research_ 43(Database issue):D682-9, 2015 Jan
</dd>
<dt>PhylDiag: identifying complex synteny blocks that include tandem duplications using phylogenetic gene trees.</dt>
<dd>
Lucas JM, Muffato M, Roest Crollius H. \\
_BMC bioinformatics_ 15:268, 2014 
</dd>
<dt>Genomicus: five genome browsers for comparative genomics in eukaryota.</dt>
<dd>
Louis A, Muffato M, Roest Crollius H. \\
_Nucleic acids research_ 41(Database issue):D700-5, 2013 Jan
</dd>
<dt>Genomicus: a database and a browser to study gene synteny in modern and ancestral genomes.</dt>
<dd>
Muffato M, Louis A, Poisnel CE, Roest Crollius H. \\
_Bioinformatics (Oxford, England)_ 26(8):1119-1121, 2010 Apr
</dd>
<dt>Paleogenomics in vertebrates, or the recovery of lost genomes from the mist of time.</dt>
<dd>
Muffato M, Roest Crollius H. \\
_BioEssays : news and reviews in molecular, cellular and developmental biology_ 30(2):122-134, 2008 Feb
</dd>
</dl>

## Quest for Orthologues

<dl>
<dt>The Quest for Orthologs benchmark service and consensus calls in 2020.</dt>
<dd>
Altenhoff AM _et al._ \\
_Nucleic acids research_ 48(W1):W538-W545, 2020 Jul
</dd>
<dt>Advances and Applications in the Quest for Orthologs.</dt>
<dd>
Glover N _et al._ \\
_Molecular biology and evolution_ 36(10):2157-2164, 2019 Oct
</dd>
<dt>Gearing up to handle the mosaic nature of life in the quest for orthologs.</dt>
<dd>
Forslund K, Pereira C, Capella-Gutierrez S, da Silva AS, Altenhoff A, Huerta-Cepas J, Muffato M, _et al._ \\
_Bioinformatics (Oxford, England)_ 34(2):323-329, 2018 Jan
</dd>
<dt>Standardized benchmarking in the quest for orthologs.</dt>
<dd>
Altenhoff AM _et al._ \\
_Nature methods_ 13(5):425-430, 2016 May
</dd>
<dt>Quest for Orthologs Entails Quest for Tree of Life: In Search of the Gene Stream.</dt>
<dd>
Boeckmann B, Marcet-Houben M, Rees JA, Forslund K, Huerta-Cepas J, Muffato M, _et al._ \\
_Genome biology and evolution_ 7(7):1988-1999, 2015 Jul
</dd>
<dt>Big data and other challenges in the quest for orthologs.</dt>
<dd>
Sonnhammer EL _et al._ \\
_Bioinformatics (Oxford, England)_ 30(21):2993-2998, 2014 Nov
</dd>
<dt>Toward community standards in the quest for orthologs.</dt>
<dd>
Dessimoz C _et al._ \\
_Bioinformatics (Oxford, England)_ 28(6):900-904, 2012 Mar
</dd>
</dl>

## Genome analysis

<dl>
<dt>The tuatara genome reveals ancient features of amniote evolution.</dt>
<dd>
Gemmell NJ _et al._ \\
_Nature_ 584(7821):403-409, 2020 Aug
</dd>
<dt>The comparative genomics and complex population history of Papio baboons.</dt>
<dd>
Rogers J _et al._ \\
_Science advances_ 5(1):eaau6947, 2019 Jan
</dd>
<dt>Repeat associated mechanisms of genome evolution and function revealed by the Mus caroli and Mus pahari genomes.</dt>
<dd>
Thybert D _et al._ \\
_Genome research_ 28(4):448-459, 2018 Apr
</dd>
<dt>LaGomiCs-Lagomorph Genomics Consortium: An International Collaborative Effort for Sequencing the Genomes of an Entire Mammalian Order.</dt>
<dd>
Fontanesi L _et al._ \\
_The Journal of heredity_ 107(4):295-308, 2016 Jul
</dd>
<dt>Gibbon genome and the fast karyotype evolution of small apes.</dt>
<dd>
Carbone L _et al._ \\
_Nature_ 513(7517):195-201, 2014 Sep
</dd>
<dt>The zebrafish reference genome sequence and its relationship to the human genome.</dt>
<dd>
Howe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, _et al._ \\
_Nature_ 496(7446):498-503, 2013 Apr
</dd>
<dt>Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.</dt>
<dd>
Smith JJ _et al._ \\
_Nature genetics_ 45(4):415-21, 421e1-2, 2013 Apr
</dd>
<dt>Plasticity of animal genome architecture unmasked by rapid evolution of a pelagic tunicate.</dt>
<dd>
Denoeud F _et al._ \\
_Science (New York, N.Y.)_ 330(6009):1381-1385, 2010 Dec
</dd>
</dl>

## ![icon](/assets/img/icon/ensembl.png) Ensembl and ![icon](/assets/img/icon/treefam.png) TreeFam NAR updates

<dl>
<dt>Ensembl 2020.</dt>
<dd>
Yates AD _et al._ \\
_Nucleic acids research_ 48(D1):D682-D688, 2020 Jan
</dd>
<dt>Ensembl Genomes 2020-enabling non-vertebrate genomic research.</dt>
<dd>
Howe KL _et al._ \\
_Nucleic acids research_ 48(D1):D689-D695, 2020 Jan
</dd>
<dt>Ensembl 2019.</dt>
<dd>
Cunningham F _et al._ \\
_Nucleic acids research_ 47(D1):D745-D751, 2019 Jan
</dd>
<dt>Ensembl 2018.</dt>
<dd>
Zerbino DR _et al._ \\
_Nucleic acids research_ 46(D1):D754-D761, 2018 Jan
</dd>
<dt>Ensembl 2017.</dt>
<dd>
Aken BL _et al._ \\
_Nucleic acids research_ 45(D1):D635-D642, 2017 Jan
</dd>
<dt>Ensembl 2016.</dt>
<dd>
Yates A _et al._ \\
_Nucleic acids research_ 44(D1):D710-6, 2016 Jan
</dd>
<dt>Ensembl 2015.</dt>
<dd>
Cunningham F _et al._ \\
_Nucleic acids research_ 43(Database issue):D662-9, 2015 Jan
</dd>
<dt>Ensembl 2014.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_ 42(Database issue):D749-55, 2014 Jan
</dd>
<dt>Ensembl 2013.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_ 41(Database issue):D48-55, 2013 Jan
</dd>
<dt>Ensembl 2012.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_ 40(Database issue):D84-90, 2012 Jan
</dd>
</dl>
