---
title: Publications
position: 4
---

# Publications

The complete list of publications is available at these databases:

* [![icon](/assets/img/icon/epmc.png) Europe PubMed
  Central](http://europepmc.org/search?query=AUTHORID:0000-0002-7860-3560&sortby=Date)
  ([profile](http://europepmc.org/authors/0000-0002-7860-3560)),
* [![icon](/assets/img/icon/googlescholar.png) Google Scholar](https://scholar.google.com/citations?user=CAik9V8AAAAJ),
* [![icon](/assets/img/icon/orcid.png) ORCID](https://orcid.org/0000-0002-7860-3560),
* [![icon](/assets/img/icon/scopus.png) Scopus](https://www.scopus.com/authid/detail.uri?authorId=23485883100).

Find below a selection organised by themes:

{::comment}
{:/comment}
* [![icon](/assets/img/icon/ensembl.png) Ensembl and ![icon](/assets/img/icon/treefam.png) TreeFam](#ENSEMBL)
* [![icon](/assets/img/icon/genomicus.png) Genomicus and ancestral genome reconstruction](#GENOMICUS)
* [![icon](/assets/img/icon/orthology.png) Quest for Orthologs](#QFO)
* [![icon](/assets/img/icon/zebrafish.png) Genome analysis](#GENOMES)
* [![icon](/assets/img/icon/ensembl.png) Ensembl yearly NAR updates](#ENSEMBL_NAR)

## ![icon](/assets/img/icon/ensembl.png) Ensembl and ![icon](/assets/img/icon/treefam.png) TreeFam {#ENSEMBL}

<dl>
<dt>Scripting Analyses of Genomes in Ensembl Plants.</dt>
<dd>
Contreras-Moreira B, Naamati G, Rosello M, Allen JE, Hunt SE, Muffato M, _et al._ \\
_Methods in molecular biology (Clifton, N.J.)_, 2443, pages 27-55, 2022  \\
DOI: [10.1007/978-1-0716-2067-0_2](https://doi.org/10.1007/978-1-0716-2067-0_2)
</dd>
<dt>ncRNA orthologies in the vertebrate lineage.</dt>
<dd>
Pignatelli M, Vilella AJ, Muffato M, Gordon L, White S, Flicek P, Herrero J. \\
_Database : the journal of biological databases and curation_, 2016, bav127 \\
DOI: [10.1093/database/bav127](https://doi.org/10.1093/database/bav127)
</dd>
<dt>Ensembl comparative genomics resources.</dt>
<dd>
Herrero J, Muffato M, _et al._ \\
_Database : the journal of biological databases and curation_, 2016, bav096 \\
DOI: [10.1093/database/bav096](https://doi.org/10.1093/database/bav096)
</dd>
<dt>Current Methods for Automated Filtering of Multiple Sequence Alignments Frequently Worsen Single-Gene Phylogenetic Inference.</dt>
<dd>
Tan G, Muffato M, Ledergerber C, Herrero J, Goldman N, Gil M, Dessimoz C. \\
_Systematic biology_, 64 (5), pages 778-791, 2015 Sep \\
DOI: [10.1093/sysbio/syv033](https://doi.org/10.1093/sysbio/syv033)
</dd>
<dt>TreeFam v9: a new website, more species and orthology-on-the-fly.</dt>
<dd>
Schreiber F, Patricio M, Muffato M, Pignatelli M, Bateman A. \\
_Nucleic acids research_, 42 (Database issue), pages D922-5, 2014 Jan \\
DOI: [10.1093/nar/gkt1055](https://doi.org/10.1093/nar/gkt1055)
</dd>
</dl>

## ![icon](/assets/img/icon/genomicus.png) Genomicus and ancestral genome reconstruction {#GENOMICUS}

<dl>
<dt>Reconstruction of hundreds of reference ancestral genomes across the eukaryotic kingdom.</dt>
<dd>
Muffato M, Louis A, Nguyen NTT, Lucas J, Berthelot C, Roest Crollius H. \\
_Nature ecology & evolution_, 7 (3), pages 355-366, 2023 Mar \\
DOI: [10.1038/s41559-022-01956-z](https://doi.org/10.1038/s41559-022-01956-z)
</dd>
<dt>The 3D organization of chromatin explains evolutionary fragile genomic regions.</dt>
<dd>
Berthelot C, Muffato M, Abecassis J, Roest Crollius H. \\
_Cell reports_, 10 (11), pages 1913-1924, 2015 Mar \\
DOI: [10.1016/j.celrep.2015.02.046](https://doi.org/10.1016/j.celrep.2015.02.046)
</dd>
<dt>Genomicus update 2015: KaryoView and MatrixView provide a genome-wide perspective to multispecies comparative genomics.</dt>
<dd>
Louis A, Nguyen NT, Muffato M, Roest Crollius H. \\
_Nucleic acids research_, 43 (Database issue), pages D682-9, 2015 Jan \\
DOI: [10.1093/nar/gku1112](https://doi.org/10.1093/nar/gku1112)
</dd>
<dt>PhylDiag: identifying complex synteny blocks that include tandem duplications using phylogenetic gene trees.</dt>
<dd>
Lucas JM, Muffato M, Roest Crollius H. \\
_BMC bioinformatics_, 15, pages 268, 2014 Aug \\
DOI: [10.1186/1471-2105-15-268](https://doi.org/10.1186/1471-2105-15-268)
</dd>
<dt>Genomicus: five genome browsers for comparative genomics in eukaryota.</dt>
<dd>
Louis A, Muffato M, Roest Crollius H. \\
_Nucleic acids research_, 41 (Database issue), pages D700-5, 2013 Jan \\
DOI: [10.1093/nar/gks1156](https://doi.org/10.1093/nar/gks1156)
</dd>
<dt>Genomicus: a database and a browser to study gene synteny in modern and ancestral genomes.</dt>
<dd>
Muffato M, Louis A, Poisnel CE, Roest Crollius H. \\
_Bioinformatics (Oxford, England)_, 26 (8), pages 1119-1121, 2010 Apr \\
DOI: [10.1093/bioinformatics/btq079](https://doi.org/10.1093/bioinformatics/btq079)
</dd>
<dt>Paleogenomics in vertebrates, or the recovery of lost genomes from the mist of time.</dt>
<dd>
Muffato M, Roest Crollius H. \\
_BioEssays : news and reviews in molecular, cellular and developmental biology_, 30 (2), pages 122-134, 2008 Feb \\
DOI: [10.1002/bies.20707](https://doi.org/10.1002/bies.20707)
</dd>
</dl>

## ![icon](/assets/img/icon/orthology.png) Quest for Orthologs {#QFO}

<dl>
<dt>Ten Years of Collaborative Progress in the Quest for Orthologs.</dt>
<dd>
Linard B _et al._ \\
_Molecular biology and evolution_, 38 (8), pages 3033-3045, 2021 Jul \\
DOI: [10.1093/molbev/msab098](https://doi.org/10.1093/molbev/msab098)
</dd>
<dt>The Quest for Orthologs benchmark service and consensus calls in 2020.</dt>
<dd>
Altenhoff AM _et al._ \\
_Nucleic acids research_, 48 (W1), pages W538-W545, 2020 Jul \\
DOI: [10.1093/nar/gkaa308](https://doi.org/10.1093/nar/gkaa308)
</dd>
<dt>Advances and Applications in the Quest for Orthologs.</dt>
<dd>
Glover N _et al._ \\
_Molecular biology and evolution_, 36 (10), pages 2157-2164, 2019 Oct \\
DOI: [10.1093/molbev/msz150](https://doi.org/10.1093/molbev/msz150)
</dd>
<dt>Gearing up to handle the mosaic nature of life in the quest for orthologs.</dt>
<dd>
Forslund K, Pereira C, Capella-Gutierrez S, da Silva AS, Altenhoff A, Huerta-Cepas J, Muffato M, _et al._ \\
_Bioinformatics (Oxford, England)_, 34 (2), pages 323-329, 2018 Jan \\
DOI: [10.1093/bioinformatics/btx542](https://doi.org/10.1093/bioinformatics/btx542)
</dd>
<dt>Standardized benchmarking in the quest for orthologs.</dt>
<dd>
Altenhoff AM _et al._ \\
_Nature methods_, 13 (5), pages 425-430, 2016 May \\
DOI: [10.1038/nmeth.3830](https://doi.org/10.1038/nmeth.3830)
</dd>
<dt>Quest for Orthologs Entails Quest for Tree of Life: In Search of the Gene Stream.</dt>
<dd>
Boeckmann B, Marcet-Houben M, Rees JA, Forslund K, Huerta-Cepas J, Muffato M, _et al._ \\
_Genome biology and evolution_, 7 (7), pages 1988-1999, 2015 Jul \\
DOI: [10.1093/gbe/evv121](https://doi.org/10.1093/gbe/evv121)
</dd>
<dt>Big data and other challenges in the quest for orthologs.</dt>
<dd>
Sonnhammer EL _et al._ \\
_Bioinformatics (Oxford, England)_, 30 (21), pages 2993-2998, 2014 Nov \\
DOI: [10.1093/bioinformatics/btu492](https://doi.org/10.1093/bioinformatics/btu492)
</dd>
<dt>Toward community standards in the quest for orthologs.</dt>
<dd>
Dessimoz C _et al._ \\
_Bioinformatics (Oxford, England)_, 28 (6), pages 900-904, 2012 Mar \\
DOI: [10.1093/bioinformatics/bts050](https://doi.org/10.1093/bioinformatics/bts050)
</dd>
</dl>

## ![icon](/assets/img/icon/zebrafish.png) Genome analysis {#GENOMES}

<dl>
<dt>Hagfish genome elucidates vertebrate whole-genome duplication events and their evolutionary consequences.</dt>
<dd>
Yu D, Ren Y, Uesaka M, Beavan AJS, Muffato M, _et al._ \\
_Nature ecology & evolution_ \\
DOI: [10.1038/s41559-023-02299-z](https://doi.org/10.1038/s41559-023-02299-z)
</dd>
<dt>The tuatara genome reveals ancient features of amniote evolution.</dt>
<dd>
Gemmell NJ _et al._ \\
_Nature_, 584 (7821), pages 403-409, 2020 Aug \\
DOI: [10.1038/s41586-020-2561-9](https://doi.org/10.1038/s41586-020-2561-9)
</dd>
<dt>The comparative genomics and complex population history of <i>Papio</i> baboons.</dt>
<dd>
Rogers J _et al._ \\
_Science advances_, 5 (1), pages eaau6947, 2019 Jan \\
DOI: [10.1126/sciadv.aau6947](https://doi.org/10.1126/sciadv.aau6947)
</dd>
<dt>Repeat associated mechanisms of genome evolution and function revealed by the <i>Mus caroli</i> and <i>Mus pahari</i> genomes.</dt>
<dd>
Thybert D _et al._ \\
_Genome research_, 28 (4), pages 448-459, 2018 Apr \\
DOI: [10.1101/gr.234096.117](https://doi.org/10.1101/gr.234096.117)
</dd>
<dt>LaGomiCs-Lagomorph Genomics Consortium: An International Collaborative Effort for Sequencing the Genomes of an Entire Mammalian Order.</dt>
<dd>
Fontanesi L _et al._ \\
_The Journal of heredity_, 107 (4), pages 295-308, 2016 Jul \\
DOI: [10.1093/jhered/esw010](https://doi.org/10.1093/jhered/esw010)
</dd>
<dt>Gibbon genome and the fast karyotype evolution of small apes.</dt>
<dd>
Carbone L _et al._ \\
_Nature_, 513 (7517), pages 195-201, 2014 Sep \\
DOI: [10.1038/nature13679](https://doi.org/10.1038/nature13679)
</dd>
<dt>The zebrafish reference genome sequence and its relationship to the human genome.</dt>
<dd>
Howe K, Clark MD, Torroja CF, Torrance J, Berthelot C, Muffato M, _et al._ \\
_Nature_, 496 (7446), pages 498-503, 2013 Apr \\
DOI: [10.1038/nature12111](https://doi.org/10.1038/nature12111)
</dd>
<dt>Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.</dt>
<dd>
Smith JJ _et al._ \\
_Nature genetics_, 45 (4), pages 415-21, 421e1-2, 2013 Apr \\
DOI: [10.1038/ng.2568](https://doi.org/10.1038/ng.2568)
</dd>
<dt>Plasticity of animal genome architecture unmasked by rapid evolution of a pelagic tunicate.</dt>
<dd>
Denoeud F _et al._ \\
_Science (New York, N.Y.)_, 330 (6009), pages 1381-1385, 2010 Dec \\
DOI: [10.1126/science.1194167](https://doi.org/10.1126/science.1194167)
</dd>
</dl>

## ![icon](/assets/img/icon/ensembl.png) Ensembl yearly NAR updates {#ENSEMBL_NAR}

<dl>
<dt>Ensembl Genomes 2022: an expanding genome resource for non-vertebrates.</dt>
<dd>
Yates AD _et al._ \\
_Nucleic acids research_, 50 (D1), pages D996-D1003, 2022 Jan \\
DOI: [10.1093/nar/gkab1007](https://doi.org/10.1093/nar/gkab1007)
</dd>
<dt>Ensembl 2022.</dt>
<dd>
Cunningham F _et al._ \\
_Nucleic acids research_, 50 (D1), pages D988-D995, 2022 Jan \\
DOI: [10.1093/nar/gkab1049](https://doi.org/10.1093/nar/gkab1049)
</dd>
<dt>Ensembl 2021.</dt>
<dd>
Howe KL _et al._ \\
_Nucleic acids research_, 49 (D1), pages D884-D891, 2021 Jan \\
DOI: [10.1093/nar/gkaa942](https://doi.org/10.1093/nar/gkaa942)
</dd>
<dt>Ensembl 2020.</dt>
<dd>
Yates AD _et al._ \\
_Nucleic acids research_, 48 (D1), pages D682-D688, 2020 Jan \\
DOI: [10.1093/nar/gkz966](https://doi.org/10.1093/nar/gkz966)
</dd>
<dt>Ensembl Genomes 2020-enabling non-vertebrate genomic research.</dt>
<dd>
Howe KL _et al._ \\
_Nucleic acids research_, 48 (D1), pages D689-D695, 2020 Jan \\
DOI: [10.1093/nar/gkz890](https://doi.org/10.1093/nar/gkz890)
</dd>
<dt>Ensembl 2019.</dt>
<dd>
Cunningham F _et al._ \\
_Nucleic acids research_, 47 (D1), pages D745-D751, 2019 Jan \\
DOI: [10.1093/nar/gky1113](https://doi.org/10.1093/nar/gky1113)
</dd>
<dt>Ensembl 2018.</dt>
<dd>
Zerbino DR _et al._ \\
_Nucleic acids research_, 46 (D1), pages D754-D761, 2018 Jan \\
DOI: [10.1093/nar/gkx1098](https://doi.org/10.1093/nar/gkx1098)
</dd>
<dt>Ensembl 2017.</dt>
<dd>
Aken BL _et al._ \\
_Nucleic acids research_, 45 (D1), pages D635-D642, 2017 Jan \\
DOI: [10.1093/nar/gkw1104](https://doi.org/10.1093/nar/gkw1104)
</dd>
<dt>Ensembl 2016.</dt>
<dd>
Yates A _et al._ \\
_Nucleic acids research_, 44 (D1), pages D710-6, 2016 Jan \\
DOI: [10.1093/nar/gkv1157](https://doi.org/10.1093/nar/gkv1157)
</dd>
<dt>Ensembl 2015.</dt>
<dd>
Cunningham F _et al._ \\
_Nucleic acids research_, 43 (Database issue), pages D662-9, 2015 Jan \\
DOI: [10.1093/nar/gku1010](https://doi.org/10.1093/nar/gku1010)
</dd>
<dt>Ensembl 2014.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_, 42 (Database issue), pages D749-55, 2014 Jan \\
DOI: [10.1093/nar/gkt1196](https://doi.org/10.1093/nar/gkt1196)
</dd>
<dt>Ensembl 2013.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_, 41 (Database issue), pages D48-55, 2013 Jan \\
DOI: [10.1093/nar/gks1236](https://doi.org/10.1093/nar/gks1236)
</dd>
<dt>Ensembl 2012.</dt>
<dd>
Flicek P _et al._ \\
_Nucleic acids research_, 40 (Database issue), pages D84-90, 2012 Jan \\
DOI: [10.1093/nar/gkr991](https://doi.org/10.1093/nar/gkr991)
</dd>
</dl>

